tests/product/yarn_slider/test_slider_installation.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Product tests for installing slider itself
+"""
+
+import json
+import os
+
+from tests.product.base_product_case import BaseProductTestCase
+from tests.product.constants import LOCAL_RESOURCES_DIR
+
+from prestoadmin.slider.config import SLIDER_CONFIG_PATH, HOST, DIR, \
+    SLIDER_USER, _SLIDER_CONFIG
+
+SLIDER_DIST_FILENAME = 'slider-assembly-0.80.0-incubating-all.tar.gz'
+
+
+class TestSliderInstallation(BaseProductTestCase):
+    def setUp(self):
+        super(TestSliderInstallation, self).setUp()
+        # For the moment, we still need to topology loaded. This will change
```

Comment:
Out of date comment; removed. This actually will run without the topology loaded, so I switched it to a PA_ONLY_CLUSTER already.